### PR TITLE
Fix Doc Links

### PR DESCRIPTION
**Pull request by danny-andrews**
## What I did

Fixed doc links

## How to test

Click the links

### CODE REVIEW

File: packages/react-storybook/README.md [hunk 1]
```diff
@@ -27,6 +27,6 @@ Here are some featured storybooks that you can reference to see how Storybook wo
 
 ## Docs
 
-* [Basics](https://getstorybook.io/docs/basics/introduction)
-* [Configurations](https://getstorybook.io/docs/configurations/default-config)
-* [Addons](https://getstorybook.io/docs/addons/introduction)
+* [Basics](https://getstorybook.io/docs/react-storybook/basics/introduction)
```
Comment (ndelangen) [inline packages/react-storybook/README.md]: Can you change these links to point to the new website instead? https://storybooks.js.org ?

**Review by ndelangen — CHANGES_REQUESTED** [review]

**Comment by ndelangen** [comment]
Thanks for this PR! @danny-andrews, if you can change the links to point to the new website, I can merge this!

**Comment by danny-andrews** [comment]
@ndelangen Done! I can squash if needed.

**Review by ndelangen — APPROVED** [review]

**Comment by ndelangen** [comment]
Thank you @danny-andrews, I hope to see more PR's by you in the future! 🙇

**Pull request merged**